examples/latency/index-async-await.js

```diff
@@ -130,7 +130,7 @@ async function main() {
         string: 'api-key',
         boolean: 'drop-all',
         default: {
-            'addr': 'http://localhost:8080'
+            'addr': 'http://localhost:8180'
```

Comment:
The default for users in docs should be 8080. 8180 is just in cases we use port offset. If we are not teaching how to use offsets here, we shouldn't change it. Could cause too much confusion for new comers.

Reply:
Sounds good @MichelDiz - I changed the readme in the example to not use a port offset.